koji_containerbuild/plugins/builder_containerbuild.py

```diff
@@ -49,7 +49,7 @@
 except ImportError:
     from osbs.exceptions import OsbsValidationException as OsbsOrchestratorNotEnabled
 try:
-    from osbs.utils import split_module_spec
+    from osbs.repo_utils import ModuleSpec
```

Comment:
Do we need to fall back to trying the older method for compatibility?

Reply:
Would it be better to check for the OSBS version available? Flatpak support was added in OSBS 2536f24, which was released in 0.43 [1].

We would use distutils.LooseVersion here since we use `major.minor.dev` for development versions between reelases.

[1] https://github.com/projectatomic/osbs-client/releases/tag/0.43

Reply:
That would be fine with me.